src/sentry/workflow_engine/types.py

```diff
@@ -142,3 +142,9 @@ class DetectorSettings:
     handler: type[DetectorHandler] | None = None
     validator: type[BaseDetectorTypeValidator] | None = None
     config_schema: dict[str, Any] = field(default_factory=dict)
+
+
+class WorkflowEngineObjectStatus(IntEnum):
```

Comment:
will it be confusing if this is different from `ObjectStatus`?

https://github.com/getsentry/sentry/blob/8e1cbf293295049dbb81002b1869074f79f88ba8/src/sentry/constants.py#L476-L482

Reply:
😭 I can't think of another name, we're not using the `HIDDEN` or `DISABLED` statuses which is why I assume we made a new enum.

Reply:
could we reuse that larger shared model? part of why i had initially made a separate one was incase we wanted to add more status / fields. if we don't want to do that now, we could probably simplify down to reusing the common one? 🤔

Reply:
Yeah we definitely can, I'll update it